src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -291,8 +292,14 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
                           return false;
                         });
 
+                        const canEditEmbargoedDegreeFile = !!user?.isEmbargoThesisCurator;
+                        const canEditNonEmbargoedDegreeFile =
+                          file.type === 'PublishedFile' ? userCanUnpublishRegistration(values) : canEditFiles;
+
                         const isEmbargoedDegreeFile = isProtectedDegree && isEmbargoed(file.embargoDate);
-                        const canEditThisFile = isEmbargoedDegreeFile ? !!user?.isEmbargoThesisCurator : canEditFiles;
+                        const canEditThisFile = isEmbargoedDegreeFile
+                          ? canEditEmbargoedDegreeFile
+                          : canEditNonEmbargoedDegreeFile;
```

Comment:
Alt dette kunne vært satt i èn ternary, men det er veldig viktig at kun embargo-thesis-curator kan endre på embargo-degree-filer. Publishing-curator-rollen skal ikke gi aksess til dette. Så for leselighetens skyld har jeg dratt det ut i consts.